pkg/cmd/log.go

```diff
@@ -129,7 +134,13 @@ func (o *logCmdOptions) run(cmd *cobra.Command, args []string) error {
 			// Found the running integration so step over to scraping its pod log
 			//
 			fmt.Fprintln(cmd.OutOrStdout(), "Integration '"+integrationID+"' is now running. Showing log ...")
-			if err := k8slog.Print(o.Context, cmd, c, &integration, cmd.OutOrStdout()); err != nil {
+			var tailLines *int64
+			if cmd.Flags().Lookup("tail-lines").Changed {
+				tailLines = &o.TailLines
+			} else if o.Tail {
+				tailLines = &[]int64{10}[0]
+			}
```

Comment:
When `o.TailLines` is explicitly defaulted to `10`, do we really need this `if`?  Also, if we do, you should use `pointer.Into64(10)` here:
```
tailLines = &[]int64{10}[0]
```

Reply:
Good catch, thx. Fixed